src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerOptions.cs

```diff
@@ -36,6 +37,11 @@ public class KestrelServerOptions
         public SchedulingMode ApplicationSchedulingMode { get; set; } = SchedulingMode.Default;
 
         /// <summary>
+        /// Gets or sets a value that controls whether synchronous IO is allowed for the <see cref="HttpContext.Request"/> and <see cref="HttpContext.Response"/> 
```

Comment:
I changed the doc comments to reference HttpContext instead of IHttpRequest/ResponseFeature. The problem is if I reference HttpContext.Request/Response.**Body**, I get the following error:

```
KestrelServerOptions.cs(40,101): error CS1574: XML comment has cref attribute 'Body' that could not be resolved [D:\dev\aspnet\KestrelHttpServer\src\Microsoft.AspNetCore.Server.Kestrel.Core\Microsoft.AspNetCore.Server.Kestrel.Core.csproj]
KestrelServerOptions.cs(40,144): error CS1574: XML comment has cref attribute 'Body' that could not be resolved [D:\dev\aspnet\KestrelHttpServer\src\Microsoft.AspNetCore.Server.Kestrel.Core\Microsoft.AspNetCore.Server.Kestrel.Core.csproj]
```

Any ideas why that is?